mapper/orm/src/main/java/org/hibernate/search/mapper/orm/model/impl/HibernateOrmPropertyModel.java

```diff
@@ -16,12 +17,15 @@
 
 import org.hibernate.AssertionFailure;
 import org.hibernate.annotations.common.reflection.XProperty;
+import org.hibernate.search.mapper.orm.logging.impl.Log;
 import org.hibernate.search.mapper.pojo.model.spi.PojoGenericTypeModel;
 import org.hibernate.search.mapper.pojo.model.spi.PojoPropertyModel;
 import org.hibernate.search.mapper.pojo.model.spi.PropertyHandle;
-import org.hibernate.search.util.SearchException;
+import org.hibernate.search.util.impl.common.LoggerFactory;
 
-class HibernateOrmPropertyModel<T> implements PojoPropertyModel<T> {
+public class HibernateOrmPropertyModel<T> implements PojoPropertyModel<T> {
```

Comment:
Making this class public looks unnecessary?

Reply:
You're right. I'll remove them.